runtime/src/main/kotlin/io/github/apl_cornell/viaduct/runtime/Boxed.kt

```diff
@@ -1,13 +1,13 @@
 package io.github.apl_cornell.viaduct.runtime
 
-abstract class Boxed {
-    abstract var temporary: Any
+/** Represents a mutable cell. */
+class Boxed<T>(var value: T) {
```

Comment:
Mark this as private.

Reply:
Philisophical question: What's the point of having a private variable with set() and get() to begin with? 
Forgot to bring this up during our meeting but I'm having trouble understanding what's wrong with just exposing the value

Reply:
Forgot to bring this up during our meeting, but why use a private variable with get() set() at all? If the only aim is to box up a value, the public variable alone seems enough

Reply:
Doing `.value` is annoying? There isn't a great reason to use `get()` and `set()` I guess, but it's fine.